modules/oxide-components/src/main/ts/bespoke/tinyai/tag/Tag.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import { Fun } from '@ephox/katamari';
+import type { Meta, StoryObj } from '@storybook/react-vite';
+
+import { Tag } from './Tag';
+
+const meta = {
+  title: 'bespoke/tinyai/Tag',
+  component: Tag,
+  parameters: {
+    // Optional parameter to center the component in the Canvas. More info: https://storybook.js.org/docs/configure/story-layout
+    layout: 'centered',
+  },
+  // This component will have an automatically generated Autodocs entry: https://storybook.js.org/docs/writing-docs/autodocs
+  tags: [ 'autodocs', 'skip-visual-testing' ],
+} satisfies Meta<typeof Tag>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+// More on writing stories with args: https://storybook.js.org/docs/writing-stories/args
```

Comment:
Do we need these doc reference comments there will be a lot if we link to every property.